lib/Service/PhishingDetection/LinkCheck.php

```diff
@@ -40,6 +40,13 @@ private function getInnerText(\DOMElement $node) : string {
 		return $innerText;
 	}
 
+	private function parse(string $url): string {
+		if(strpos($url, "://") === false) {
```

Comment:
The regex pattern seem to only check for "https://" and NOT "http://" so adding "http://" to a URL that is missing the scheme would produce a non matching result, stripping of the scheme for all links might be better

Reply:
The regex matches 
> URLs -- http, https, and naked domains like "example.com".  
 but also this method is called later than the regex it adds http:// ( the protocol here doesn't really matter) it's only added so [parse_url](https://www.php.net/manual/en/function.parse-url.php) can work properly